### PR TITLE
VRE chapter 31 military history

### PR DESCRIPTION
**Pull request by cehsu**
These changes set up the military history chapter for chapter 31.
606dfbc94441ee52e01dad8f9a1edc14 review vetsgov-internal_employment_vocational-rehab-and-employment_application_chapter31_military-history

### CODE REVIEW

**Comment by jbalboni** [comment]
@cehsu Can you add a screenshot?

**Review by rianfowler — APPROVED** [review]
LGTM

**Comment by cehsu** [comment]
Thanks @jbalboni, screenshot added.

File: src/js/vre/chapter31/config/form.js [hunk 1]
```diff
@@ -60,14 +126,20 @@ const formConfig = {
       title: 'Military History',
       pages: {
         militaryHistory: {
-          path: 'military-information',
-          title: 'Military Information',
+          path: 'military-history',
+          title: 'Military History',
+          uiSchema: {
+            serviceHistory: serviceHistoryUI,
```
Comment (jbalboni) [inline src/js/vre/chapter31/config/form.js]: Unless you are using these more than once, I think it's easier to read if they're in line here (not defined up above in a variable). That way you read them the context of the page config they're used on.

File: test/vre/chapter31/config/militaryhistory.unit.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        formData={{}}
+        onSubmit={onSubmit}
+        uiSchema={uiSchema}/>
+    );
+
+    form.find('form').simulate('submit');
+    expect(form.find('.usa-input-error').length).to.equal(0);
+
+    expect(onSubmit.called).to.be.true;
+  });
```
Comment (jbalboni) [inline test/vre/chapter31/config/militaryhistory.unit.spec.jsx]: I've been trying to write tests that add an item when the page has an array field.

**Review by jbalboni — APPROVED** [review]

File: src/js/vre/chapter31/config/form.js (hunk 1 above)
Comment (cehsu) [reply src/js/vre/chapter31/config/form.js]: Good point, relocated.

File: test/vre/chapter31/config/militaryhistory.unit.spec.jsx (hunk 2 above)
Comment (cehsu) [reply test/vre/chapter31/config/militaryhistory.unit.spec.jsx]: Thanks for pointing that out, added.

**Pull request merged**